### ISSUE TITLE
Persistant popup settings memory on all browsers

### ISSUE THREAD

**Issue by hugolpz** [lingua-libre/SignIt#87]
See also #56

**Comment by kabir-afk** [comment]
@hugolpz I wanted to confirm one thing ? I resolved it partially as mentioned in above referenced PR as I fixed how `storeParam` was being executed. But we'd still want the user's preferred setings to persist even if he open a new tab or reloads the page right ?Because my solution works as long as service_worker is alive and that too for the current tab. They reset when reloaded or a new tab is opened. I would say it makes the settings tab functional , but partially.

**Comment by hugolpz** [comment]
@kabir-afk Correct. Ideally settings are persistent even if we close the current tab and open a new one. Or even when we close the browser and whole computer.

**Comment by kabir-afk** [comment]
I'll see what I can do
